src/main/resources/com/rultor/agents/daemons/end.sh

```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/env bash
+#@todo #754:30min Remove the deprecation code from the shell.
+# Move the code to Java. May be
```

Comment:
@sebing @mkordas I don't think a design solution is even needed here fortunately :)
Look @sebing , just give this file a glance https://github.com/yegor256/rultor/blob/master/src/test/java/com/rultor/profiles/GithubProfileITCase.java.
It clearly shows how to extract the `script` section from the Profile, this should give you all you need to figure out the `image` on the Java end, shouldn't it ? :)